support/linux/install_dev_9_linux.sh

```diff
@@ -45,4 +45,4 @@ fi
 
 sudo sh /tmp/install.sh
 sudo hab install core/busybox-static core/hab-studio
-sudo rm -rf /tmp/install.sh
+sudo rm -f /tmp/install.sh
```

Comment:
Per https://github.com/habitat-sh/habitat/pull/6114/files/72d9ce5e429e6e6721f8c7ac0c35f1da0a035c34#diff-4e85b8e641a4a00ebf0df4a2b2887219, no net changes to this file, please. https://github.com/habitat-sh/habitat/pull/6113 has been reviewed and is waiting on your next steps, to please address this in that PR.

Reply:
I swear there are gremlins afoot.  I distinctly remember fixing this.

:+1: fixed.